client/node.go

```diff
@@ -322,6 +320,13 @@ func (n *NodeClient) NetworkGetPublicConfig(ctx context.Context) (cfg pkg.Public
 	return
 }
 
+func (n *NodeClient) NetworkGetNodeFeatures(ctx context.Context) (feat pkg.NodeFeatures, err error) {
+	const cmd = "zos.network.node_features_get"
```

Comment:
Why on the zos.network? I assumed it will be on `zos.system.node_features`

Reply:
since this is related to netowrkd-light I assumed it would be on network, but can move it to be system.node_features

Reply:
My reasoning is as a user I should ask the node what version do you have? (which is `zos.system.version`) or what features are enabled on the node so `zos.system.node_features_get`